pom.xml

```diff
@@ -9,37 +9,39 @@
 	</parent>
 
 	<artifactId>charges-data-api</artifactId>
+
+
 	<version>unversioned</version>
 	<name>charges-data-api</name>
 	<description>Charges Data Api</description>
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<java.version>11</java.version>
+		<java.version>21</java.version>
 
 		<main.class>uk.gov.companieshouse.charges.data.ChargesDataApiApplication</main.class>
 
 		<maven.compiler.source>${java.version}</maven.compiler.source>
 		<maven.compiler.target>${java.version}</maven.compiler.target>
 		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
-		<spring.boot.version>2.6.4</spring.boot.version>
+		<spring.boot.version>3.2.3</spring.boot.version>
 		<assertj-core.version>3.22.0</assertj-core.version>
-		<test-containers.version>1.16.3</test-containers.version>
-		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
-		<maven-build-helper-plugin.version>3.2.0</maven-build-helper-plugin.version>
-		<maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version>
+		<test-containers.version>1.19.7</test-containers.version>
+		<maven-surefire-plugin.version>3.2.5</maven-surefire-plugin.version>
+		<maven-build-helper-plugin.version>3.5.0</maven-build-helper-plugin.version>
+		<maven-failsafe-plugin.version>3.2.5</maven-failsafe-plugin.version>
 		<docker-maven-plugin.version>1.0.0</docker-maven-plugin.version>
-		<structured-logging.version>1.9.12</structured-logging.version>
-		<api-sdk-manager-java-library.version>1.0.4</api-sdk-manager-java-library.version>
-		<api-helper-java-library.version>1.4.5</api-helper-java-library.version>
-		<cucumber-java.version>7.2.3</cucumber-java.version>
-		<jib-maven-plugin.version>3.1.1</jib-maven-plugin.version>
+		<structured-logging.version>3.0.3</structured-logging.version>
+		<api-sdk-manager-java-library.version>3.0.5</api-sdk-manager-java-library.version>
+		<api-helper-java-library.version>3.0.1</api-helper-java-library.version>
+		<cucumber-java.version>7.15.0</cucumber-java.version>
+		<jib-maven-plugin.version>3.4.0</jib-maven-plugin.version>
 		<skip.integration.tests>false</skip.integration.tests>
 		<skip.unit.tests>false</skip.unit.tests>
-		<private-api-sdk-java.version>2.0.181</private-api-sdk-java.version>
+		<private-api-sdk-java.version>4.0.76</private-api-sdk-java.version>
 		<io-cucumber.version>7.2.3</io-cucumber.version>
-		<wiremock.standalone.version>2.32.0</wiremock.standalone.version>
+		<wiremock.standalone.version>3.0.1</wiremock.standalone.version>
 		<maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
 
 		<!--sonar configuration-->
```

Comment:
3.11.0.3922

Reply:
is this comment for the wiremock.standalone.version?

Reply:
no the sonar-maven-plugin

Reply:
done